### DISCUSSION TITLE
Space to select?

### DISCUSSION THREAD

**Discussion by OpenAMOC** [sanni/cartreader#937] [Need help]
When trying to dump a GBA title using serial monitor, I am asked to press space to select. I'm probably missing something obvious but I when I press space in the window, or type space in the text field and send, nothing happens. 

I have tried solving myself, used different games, different computers, and attempted every combination of sending " ", Serial.print(" "), \t instead of a space but no luck. Any help or advice much appreciated!

![Screenshot 2024-04-27 125810](https://github.com/sanni/cartreader/assets/95543179/0a7b18cf-33ec-411b-bb7b-fdd9f855d532)

**Comment by Ancyker** [comment]
I haven't used the serial monitor feature, but try using a real serial terminal and see if that helps. Something like [PuTTY](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html) (if you are on Windows) should work.

**Reply by OpenAMOC** [reply]
An interesting idea, thank you! Yes using PuTTY is much more familiar, don't know why it didn't come to mind. Unfortunatly, I encounter the same issue! I press space to no response, though other required inputs like numbers, u and d work fine.

![image](https://github.com/sanni/cartreader/assets/95543179/d6aa1dfa-c291-47b9-9754-d83d922ae3c6)

**Comment by OpenAMOC** [comment]
Can't say I've found a fix but instead a way around it. Simply edited the code to use a different input, I chose 1, as pictured.

![image](https://github.com/sanni/cartreader/assets/95543179/85f99d87-7f95-4861-ab6f-71ec67a25a29)
